test/e2e/app-dir/hooks/hooks.test.ts

```diff
@@ -60,6 +60,9 @@ createNextDescribe(
         expect($('#params-second').text()).toBe('other value')
         expect($('#params-third').text()).toBe('')
         expect($('#params-not-real').text()).toBe('N/A')
+
+        // should not have noindex meta tag as it's erroring on purpose, for nextjs internal use only
+        expect($('meta[name=noindex]').length).toBe(0)
```

Comment:
won't this be true regardless of the changes here since this page doesn't have any errors?

Reply:
Nice catch, I separate them into dynamic and static-opt case, so for static-opt case in production it won't have this meta tag. Also a typo on the property 🙈